src/ClusterBootstrap/cloud_init_deploy.py

```diff
@@ -500,23 +500,29 @@ def add_implied_services_based_on_config(config):
     additional_svcs = []
     if config.get("job-manager", {}).get("launcher", "") == "controller":
         additional_svcs.append("launcher")
-        print(additional_svcs)
     return additional_svcs
 
 
-def render_infra_node_specific(config, args):
-    assert config["priority"] in ["regular", "low"]
-
-    config["etcd_endpoints"] = ",".join(
-        ["https://" + x + ":" + config["etcd3port1"] for x in config["etcd_node"]]).replace("/", "\\/")
-
+def nfs_client_config(config):
     with open("./deploy/storage/auto_share/mounting.yaml", 'r') as mf:
         mounting = yaml.safe_load(mf)
     config["mount_and_link"] = []
     for mnt_itm in mounting.values():
         for fs in mnt_itm["fileshares"]:
             config["mount_and_link"] += fs["remote_mount_path"],
+    return config
 
+
+def escaped_etcd_end_point_and_k8s_api_server(config):
+    config["escaped_etcd_endpoints"] = ",".join(
```

Comment:
Can we stick with `etcd_endpoints` and `api_servers`? Why adding `escaped`?

Reply:
to avoid pop-out-to-recover, and overriding issue when only certain commands are executed/re-executed.

Reply:
Can you give an example of such issue?

Reply:
renderspecific nfs after render worker etc.

Reply:
actually previously there are more "\/" in the escaped url in NFS's cloudinit file before. Using an independent variable is safer, and make the diff between escaped url v.s. env var name clear